e2e/testcases/helm_sync_test.go

```diff
@@ -57,6 +63,58 @@ func TestPublicHelm(t *testing.T) {
 	}
 }
 
+// TestHelmNamespaceRepo verifies RepoSync does not sync the helm chart with cluster-scoped resources. It also verifies that RepoSync can successfully
+// sync the namespace scoped resources, and assign the RepoSync namespace to these resources.
+// This test will work only with following pre-requisites:
+// Google service account `e2e-test-ar-reader@stolos-dev.iam.gserviceaccount.com` is created with `roles/artifactregistry.reader` for accessing images in Artifact Registry.
+// A JSON key file is generated for this service account and stored in Secret Manager
+func TestHelmNamespaceRepo(t *testing.T) {
+	repoSyncNN := nomostest.RepoSyncNN(testNs, "rs-test")
+	nt := nomostest.New(t, ntopts.SkipMonoRepo,
+		ntopts.NamespaceRepo(repoSyncNN.Namespace, repoSyncNN.Name))
```

Comment:
This test should require running on GKE because it accesses the private AR registry.

Reply:
Done.